xcube/core/compute.py

```diff
@@ -53,6 +53,50 @@ def compute_cube(cube_func: CubeFunc,
     from variables named *input_var_names* contained in zero, one, or more
     input data cubes in *input_cubes* using a cube factory function *cube_func*.
 
+    For a more detailed description of the function usage,
+    please refer to :func:compute_dataset.
+
+    :param cube_func: The cube factory function.
+    :param input_cubes: An optional sequence of input cube datasets, must be provided if *input_cube_schema* is not.
+    :param input_cube_schema: An optional input cube schema, must be provided if *input_cubes* is not.
+    :param input_var_names: A sequence of variable names
+    :param input_params: Optional dictionary with processing parameters passed to *cube_func*.
+    :param output_var_name: Optional name of the output variable, defaults to ``'output'``.
+    :param output_var_dtype: Optional numpy datatype of the output variable, defaults to ``'float32'``.
+    :param output_var_attrs: Optional metadata attributes for the output variable.
+    :param vectorize: Whether all *input_cubes* have the same variables which are concatenated and passed as vectors
```

Comment:
The name 'vectorize' sounds like you toggle an action/ an option, but the decription sounds like you are describing a cube property.